<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirspyHFGui</class>
 <widget class="QWidget" name="AirspyHFGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>160</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AirspyHF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop acquisition</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="toolTip">
            <string>I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Liberation Mono</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Tuner center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="text">
        <string> kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="freqRightlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loCorrectionLayout">
     <item>
      <widget class="QLabel" name="LOppmLabel">
       <property name="text">
        <string>LO ppm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="LOppm">
       <property name="toolTip">
        <string>Local Oscillator ppm correction</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LOppmText">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>LO correction value (ppm)</string>
       </property>
       <property name="text">
        <string>-00.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetLOppm">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Rest LO ppm correction</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="dcOffset">
       <property name="toolTip">
        <string>DC component block (internal DSP)</string>
       </property>
       <property name="text">
        <string>DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="iqImbalance">
       <property name="toolTip">
        <string>IQ imbalance correction (internal DSP)</string>
       </property>
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="bandLabel">
       <property name="text">
        <string>Band</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="band">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Band select</string>
       </property>
       <item>
        <property name="text">
         <string>HF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VHF</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sampleRateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>S/R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sampleRate">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Device sample rate in MS/s</string>
       </property>
       <item>
        <property name="text">
         <string>0000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rateLabel">
       <property name="text">
        <string>k</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="fcPosRightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_decim">
       <property name="text">
        <string>Dec</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="decim">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decimation factor</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="TransverterButton" name="transverter">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Transverter frequency translation dialog</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="agcLabel">
       <property name="text">
        <string>AGC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="agc">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>AGC mode and threshold (HF only)</string>
       </property>
       <item>
        <property name="text">
         <string>Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="lna">
       <property name="toolTip">
        <string>LNA on/off (HF only)</string>
       </property>
       <property name="text">
        <string>LNA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="attLabel">
       <property name="text">
        <string>Att</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="att">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Attenuator (AGC off and HF only)</string>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>42</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="attUnits">
       <property name="text">
        <string>dB</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ButtonSwitch" name="dsp">
       <property name="toolTip">
        <string>Toggle libairspyhf DSP on/off</string>
       </property>
       <property name="text">
        <string>DSP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="paddingLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransverterButton</class>
   <extends>QPushButton</extends>
   <header>gui/transverterbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
